README.it.md

```
@@ -0,0 +1,613 @@
+```json
+"baseUrl": "types",
+"typeRoots": ["types"],
+```
+
+Crea un `types/foo/index.d.ts` contenente le dichiarazioni per il tuo modulo "pippo".
+Ora dovresti essere in grado di importare `"pippo"` nel tuo codice, con i tipi riferiti alle dichiarazioni che hai appena creato.
+Poi fai una build **ed** esegui il codice per essere sicuro che le definizioni dei tipi corrispondano effettivamente a ciò che accade a runtime.
+
+Una volta che hai testato le definizioni su del codice reale, fai una [Pull Request](#fai-una-pull-request)
+e poi segui le istruzioni per [modificare un package preesistente](#modificare-un-package-preesistente] o [creare un nuovo package](#creare-un-nuovo-package).
+
```

markdownlint-cli2 (0.17.2)

Emphasis style
Expected: underscore; Actual: asterisk

(MD049, emphasis-style)